conf/nginx/templates/nginx.conf.web.template

```diff
@@ -2,6 +2,10 @@ http
 {
     # necessary to have content-type while serving files
     include /opt/zextras/common/conf/mime.types;
+    types
```

Comment:
I would move these lines in a location in order to have a more limited scope
Here should be fine
https://github.com/zextras/carbonio-proxy/blob/devel/conf/nginx/templates/nginx.conf.web.https.default.template#L108

(only affects /static requests )

location context is supported for custom mime declarations
https://nginx.org/en/docs/http/ngx_http_core_module.html#types